views/wishCardFullPage.ejs

```diff
@@ -8,6 +8,10 @@
     <!-- ** NOTE: CDN Bootstrap.js, Popper.js, and jQuery are moved to the bottom so the page loads faster-->
     <%- include('./templates/header')-%>
     <link rel="stylesheet" href="../css/wishCardFull.css">
+    <!-- jquery -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <!-- js scripts -->
+    <script src="../clientJS/wishCardFullPage.js"></script>
```

Comment:
we dont need double dots, /clientJS/ should be enough and wont cause issues in nested routes, double dots will.